File: app-components/src/prod/java/org/synsystems/onlypass/components/ProdApp.java
import org.synsystems.onlypass.components.App;
import org.synsystems.onlypass.components.AppComponent;
import org.synsystems.onlypass.components.DaggerProdAppComponent;

public class ProdApp extends App {
  @Override
  protected AppComponent createAppComponent() {
    return DaggerProdAppComponent.create();
  }
}

File: app-components/src/prod/java/org/synsystems/onlypass/components/ProdApp.kt
import org.synsystems.onlypass.components.App
import org.synsystems.onlypass.components.AppComponent
import org.synsystems.onlypass.components.DaggerProdAppComponent

class ProdApp : App() {

  @Override
  protected fun createAppComponent() = DaggerProdAppComponent.create()
}